### ISSUE TITLE
The format "Align" affects all context in editor

### ISSUE THREAD

**Issue by kitara2005** [tinymce/tinymce#4258]
I have a settings of TinyMCE 4 as follows:

```
<script type="text/javascript">
tinymce.init({
    selector: "textarea",
    plugins: [
        "advlist autolink lists link image charmap print preview anchor",
        "searchreplace visualblocks code fullscreen",
        "insertdatetime media table contextmenu paste"
    ],
    toolbar: "insertfile undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image",
	forced_root_block : 'div',
	paste_webkit_styles: "all",
    paste_retain_style_properties : "all",
    paste_enable_default_filters: false,
    paste_auto_cleanup_on_paste : false,
	
});
</script>

<form method="post" action="dump.php">
    <textarea name="content"></textarea>
</form>
```

And when I paste content to textarea (please copy the following and paste)

This
That

I select "This" and using "Align right" for "This"
=> "That" is also aligned right, too.

I find out that when I copy and paste to textarea , the content is wrapped in a div, although it's on 2 lines.

I don't know whether this phenomenon is right or not?
And anyone know that how to separate to two div after paste to textarea?

Thank you.

**Comment by fyrkant** [comment]
The problem is that it's just a `<br />` between this and that when you paste it, so they are in the same div element and will both get floated to the right. If this and that are in separate elements they can have different alignment.

**Comment by kitara2005** [comment]
@fyrkant 

Thank you for your information.
Is it phenomenon of TinyMCE?

And if I want to fix it, should I remove `<br /` before insert it to richtext, shouldn't it?

**Comment by annawolf913** [comment]
May it be that the forced_root_block doesn't work correctly? I'm working with version 4.7.4 and the forced_root_block only works with "p" and false correctly... any other tag names will be ignored. So I've checked the source code and I've found out there seems to be a typo in the implementation (editor.settings.forcedRootBlock vs. editor.settings.forced_root_block). But if I define forcedRootBlock instead of forced_root_block, the browser crashes...
